packages/app/src/systems/Transaction/component/TxInfo/TxInfo.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import { Card } from '@fuels/ui';
+import type { BaseProps } from '@fuels/ui';
+import type { ReactNode } from 'react';
+import { tv } from 'tailwind-variants';
+
+type TxInfoProps = BaseProps<{
+  name: string;
+  description?: string | null;
+  children?: ReactNode;
+}>;
+
+export function TxInfo({
+  className,
+  name,
+  description,
+  children,
+  ...props
+}: TxInfoProps) {
+  const classes = styles();
+  return (
+    <Card {...props} className={classes.root({ className })}>
+      <Card.Header className="gap-0">
+        <Card.Description className="font-medium text-sm h-6 flex items-center">
+          {name}
+        </Card.Description>
+        <Card.Description className="font-medium text-md text-current my-1 h-6 flex items-center color-2">
+          {children}
+        </Card.Description>
+        <Card.Description className="text-sm flex items-center">
+          {description || '  '}
+        </Card.Description>
```

Comment:
```suggestion
        {description &&
          <Card.Description className="text-sm flex items-center">
            {description}
          </Card.Description>
        }
```
would this be better?

Reply:
This would make the description space to be smaller the idea of the empty space it to keep the space as it's and not change the size of the card